### PR TITLE
start building opensuse

### PR DESCRIPTION
**Pull request by EnderNightLord-ChromeBook**

### CODE REVIEW

File: assets/arm/assets.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (EnderNightLord-ChromeBook) [inline assets/arm/assets.txt]: I might leave these to you i'm not sure how these work...

Comment (corbinlc) [reply assets/arm/assets.txt]: installArch.sh creates them.  They are just a list of files and timestamps.

**Comment by corbinlc** [comment]
I don't see any of the tar.gz files.  Creating those is normally the most difficult piece of the work.

File: assets/arm/assets.txt (hunk 1 above)
Comment (EnderNightLord-ChromeBook) [reply assets/arm/assets.txt]: right...

**Comment by EnderNightLord-ChromeBook** [comment]
> I don't see any of the tar.gz files. Creating those is normally the most difficult piece of the work.

I don't have the rootfs yet...

**Pull request merged**